src/tellor_disputables/data.py

```diff
@@ -49,6 +52,20 @@ def get_contract_info(chain_id: int) -> Tuple[Optional[str], Optional[str]]:
         return None, None
 
 
+def get_query_type(q: OracleQuery) -> str:
+    """Get query type from class name"""
+    return type(q).__name__
+
+
+def get_query_id(q: OracleQuery) -> str:
+    """Get query id from OracleQuery"""
+    return str(q.query_id.hex())
```

Comment:
You don't need to cast as string here since `.hex()` already returns a string.

Reply:
good point!